atomics/CMakeLists.txt

```diff
@@ -87,5 +87,7 @@ install(DIRECTORY
   PATTERN *.inc
   PATTERN *.inc_*)
 
-add_subdirectory(unit_tests)
-add_subdirectory(performance_tests)
+if(BUILD_TESTING)
```

Comment:
Please discuss the choice for the option to enable testing

Reply:
The CTest module defines a `BUILD_TESTING` but it is not obvious to me where the `include(CTest)` would occur.

Reply:
Considering https://github.com/LLNL/blt/blob/8c229991e65e7a9603c621b47cb3ba158bb7468c/cmake/BLTOptions.cmake#L16 and the other options we currently accept, is `ENABLE_TESTS` a better choice?

Reply:
And there is `DESUL_ENABLE_TESTS` as well  
https://github.com/desul/desul/blob/f63c5ee63a2f3aababaad995561d15b2b4951611/CMakeLists.txt#L14
https://github.com/desul/desul/blob/f63c5ee63a2f3aababaad995561d15b2b4951611/atomics/unit_tests/kokkos_based/CMakeLists.txt#L1